test/test_torch.py

```diff
@@ -2683,6 +2683,17 @@ def test_qtensor(self):
         # This calls _th_fill_
         # qr[0] = 8 # float asignment
         # self.assertEqual(qr.item(), 8)
+        # we can also print a qtensor
+        self.assertEqual(str(qr),
+                         "tensor(is_quantized=True, qscheme=per_tensor_affine,\n" +
```

Comment:
did you respond to the question about why we are printing both?

Reply:
see comment